Streetcode/Streetcode.WebApi/Controllers/Source/SourcesController.cs

```diff
@@ -27,23 +31,21 @@ public async Task<IActionResult> GetSubCategoriesByCategoryId([FromRoute] int id
     }
 
     [HttpPost]
-    public async Task<IActionResult> Create([FromBody] PartnerDTO partner)
+    public async Task<IActionResult> CreateCategory([FromBody] SourceLinkDTO category)
```

Comment:
As a future user of this function, I'd like more info in the method name. For ex: CreateStreetcodeCategory.